src/components/Header.js

```diff
@@ -0,0 +1,17 @@
+import { Link } from 'react-router-dom';
+
+const Header = (props) => {
+  const route = props.route; // eslint-disable-line
+  return (
```

Comment:
- [ ] Kindly implement the changes requested by the previous reviewers.

Reply:
I tried fixing the linter error but It would require me to change my code significantly and it will take much time.